src/components/Activity/activity-types.js

```diff
@@ -1,6 +1,6 @@
 import actions from '../../actions/court-action-types'
 
-import iconAnj from './assets/activity-icon-anj.svg'
```

Comment:
Let's delete this asset

Reply:
do you know which is the correct one to use? since we are using it in several types